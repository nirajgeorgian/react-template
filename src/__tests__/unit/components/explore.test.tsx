import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import HomescreenBanner from 'components/banner/homepage/index'

const ExploreComponent = (
	<HomescreenBanner
		imagesrc="http://www.hdnicewallpapers.com/Walls/Big/Beach/Beautiful_Bondi_Beach_in_Australia_2018_Travel_HD_Wallpaper.jpg"
		imagealt=""
		pretitle="Satyology Experiences"
		title="Indulge in the culture of every city like a local."
		buttontext="Experiences"
	/>
)

it('renders correctly', () => {
  const {  asFragment } = render(ExploreComponent)
    expect(asFragment()).toMatchSnapshot()
})

