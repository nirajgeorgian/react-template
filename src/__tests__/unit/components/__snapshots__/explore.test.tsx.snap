// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div>
    <img
      alt=""
      src="http://www.hdnicewallpapers.com/Walls/Big/Beach/Beautiful_Bondi_Beach_in_Australia_2018_Travel_HD_Wallpaper.jpg"
    />
  </div>
  <div>
    <h2>
      Satyology Experiences
    </h2>
    <p>
      <b>
        Indulge in the culture of every city like a local.
      </b>
    </p>
    <p>
      <button
        class="ant-btn ant-btn-background-ghost"
        type="button"
      >
        <span>
          Explore Experiences
        </span>
      </button>
    </p>
    <p />
  </div>
</DocumentFragment>
`;
